### ISSUE TITLE
Class is added that doesn't exist to <table> element

### ISSUE THREAD

**Issue by feebala** [nextbigsoundinc/stylemark#31]
I have the following in a markdown file: 

```
---
name: Striped Tables
category: Tables
---
Tables in angular/webapps should **only** be used for tabular data. 

```table-types.html
<table class="striped">
<thead>
	<tr>
		<th>Striped Table Header</th>
		<th>Header 2</th>
		<th>Actions</th>
	</tr>
</thead>

<tbody>
	<tr>
		<td>Row 1 cell 1</td>
		<td>Row 1 cell 2</td>
		<td><button>Edit</button></td>
	</tr>
	<tr>
		<td>Row 2 cell 1</td>
		<td>Row 2 cell 2</td>
		<td><button>Edit</button></td>
	</tr>
	<tr>
		<td>Row 3 cell 1</td>
		<td>Row 3 cell 2</td>
		<td><button>Edit</button></td>
	</tr>
</tbody>

</table>
```

When it generates the styleguide, the html displays like this:

```
<table class="table" class="striped">
<thead>
    <tr>
        <th>Striped Table Header</th>
        <th>Header 2</th>
        <th>Actions</th>
    </tr>
</thead>

<tbody>
    <tr>
        <td>Row 1 cell 1</td>
        <td>Row 1 cell 2</td>
        <td><button>Edit</button></td>
    </tr>
    <tr>
        <td>Row 2 cell 1</td>
        <td>Row 2 cell 2</td>
        <td><button>Edit</button></td>
    </tr>
    <tr>
        <td>Row 3 cell 1</td>
        <td>Row 3 cell 2</td>
        <td><button>Edit</button></td>
    </tr>
</tbody>

</table>
```
It's adding in a `class="table"` which isn't in the markdown file, and I have no idea why. It only does this with table elements. Any idea why?

**Comment by mpetrovich** [comment]
That class is added during the HTML generation step. Looks like it’s being applied too aggressively—we’ll fix!

**Comment by mpetrovich** [comment]
Fixed in v0.6.8.

Thanks for reporting!
